src/silx/io/nxdata/_utils.py

```diff
@@ -83,10 +90,13 @@ def get_attr_as_unicode(item, attr_name, default=None):
             # other array, most likely unicode objects
             return [element for element in attr]
     else:
+        # TODO: Fix typing
         return copy.deepcopy(attr)
 
 
-def get_uncertainties_names(group, signal_name):
+def get_uncertainties_names(
+    group: h5py.Group, signal_name: str | bytes
```

Comment:
Is `bytes` needed here?
```suggestion
    group: h5py.Group, signal_name: str
```